nixos/roles/mysql.nix

```diff
@@ -145,6 +159,12 @@ with builtins;
         sysdate-is-now             = 1
         sql_mode                   = NO_ENGINE_SUBSTITUTION
 
+        log_slow_verbosity = 'full'
+        slow_query_log = ON
+        long_query_time = 0
```

Comment:
We should use 100ms like for Postgresql to avoid logging too many small queries.

Is it easy to override these values via Nix code? Otherwise we should add options to change the query time and verbosity.

Reply:
percona would need a patch to use rfc42-style options then any of those values could be easily changed. I'm going to add that, long-term that's better than inventing our own options

Reply:
Is percona just mysql but slightly different? We could just use upstream, it has rfc42 already

Reply:
I've looked through things and it seems like the upstream module has some of the fc changes missing, most notably rootPasswordFile

Not sure but inventing an option versus going upstream and not being able to modify it for now both don't seem like good choices, which one then..

Reply:
Yeah, we should aim for unifying the percona and the upstream MySQL stuff, getting the rootPasswordFile upstream would be welcome too, but let's do that in a separate issue.

Reply:
We can override default settings per flyingcircus.roles.mysql.extraConfig or /etc/local/mysql. They appear later in the generated config file and have precedence.